app/actors/sufia/create_with_files_actor.rb

```diff
@@ -42,5 +42,36 @@ def uploaded_files
         return [] if uploaded_file_ids.empty?
         @uploaded_files ||= UploadedFile.find(uploaded_file_ids)
       end
+
+      def mark_inactive
+        return true unless Flipflop.enable_mediated_deposit?
+        curation_concern.state = inactive_uri
+        send_inactive_message
+      end
+
+      def inactive_uri
+        ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive')
+      end
+
+      def send_inactive_message
+        return true if receiving_users == []
+        receiving_users.each do |admin|
+          message = "#{user.name} created <a href='#{work_path}'>#{curation_concern.title.first}</a>"
+          ::User.batch_user.send_message(admin, message, 'New work to review')
+        end
+      end
+
+      # Gather all admin users (if the admin role is defined)
+      def receiving_users
+        receivers = []
+        ::User.find_each do |user|
```

Comment:
WIll the Sipity stuff in CC make this easier to ask what all users have a particular role?